### PR TITLE
Update SE-EDU/contributing section url in README file

### LINKED ISSUE #34: Update URL of SE-EDU / Contributing section in README file
Context:
Screenshot 2024-07-27 at 11 56 54 AM

A minor issue here. It seems that this url format is quite weird to me, although it has failed to direct me to the correct site from IntelliJ, clicking the link from the README file works perfectly. Perhaps it could be changed to a more familiar url like "https://se-education.org/#contributing" instead of "https://se-education.org#https://se-education.org/#contributing"

### PR DESCRIPTION
**Pull request by jyue487**
Closes #34

### CODE REVIEW

**Review by hjungwoo01 — APPROVED** [review]
LGTM. Updated the URL to the correct one.

**Review by aureliony — CHANGES_REQUESTED** [review]
The URL is still incorrect. It should be https://se-education.org/#contributing-to-se-edu, which automatically scrolls down to the `Contributing to SE-EDU` header.

**Comment by hjungwoo01** [comment]
> The URL is still incorrect. It should be https://se-education.org/#contributing-to-se-edu, which automatically scrolls down to the `Contributing to SE-EDU` header.

Yeah, this is better. I overlooked that. Directing the users straight to the “Contributing to SE-EDU” header is more efficient. Thanks for pointing that out!

**Comment by baskargopinath** [comment]
agreed with @aureliony @hjungwoo01

File: README.md [hunk 1]
```diff
@@ -11,4 +11,4 @@
   * It comes with a **reasonable level of user and developer documentation**.
 * It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
 * For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org/#contributing) for more info.
```
Comment (baskargopinath) [inline README.md]:
```suggestion
* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org/#contributing-to-se-edu) for more info.
```

**Comment by Carlintyj** [comment]
LGTM! I have tried the link and it scrolls to the "Contributing to SE-EDU” header section.

File: README.md (hunk 1 above)
Comment (damithc) [reply README.md]: @jyue487 Can you update the link, as suggested by @baskargopinath

**Pull request merged**